diff --git a/test/test-tcp-flags.c b/test/test-tcp-flags.c
index 04a238b2..16218a27 100644
--- a/test/test-tcp-flags.c
+++ b/test/test-tcp-flags.c
@@ -49,21 +49,6 @@ TEST_IMPL(tcp_flags) {
   r = uv_tcp_keepalive(&handle, 1, 0);
   ASSERT_EQ(r, UV_EINVAL);
 
-  r = uv_tcp_keepalive_ex(&handle, 1, 60, 60, 60);
-  ASSERT_OK(r);
-
-  r = uv_tcp_keepalive_ex(&handle, 0, 0, 0, 0);
-  ASSERT_OK(r);
-
-  r = uv_tcp_keepalive_ex(&handle, 1, 0, 10, 10);
-  ASSERT_EQ(r, UV_EINVAL);
-
-  r = uv_tcp_keepalive_ex(&handle, 1, 10, 0, 10);
-  ASSERT_EQ(r, UV_EINVAL);
-
-  r = uv_tcp_keepalive_ex(&handle, 1, 10, 10, 0);
-  ASSERT_EQ(r, UV_EINVAL);
-
   uv_close((uv_handle_t*)&handle, NULL);
 
   r = uv_run(loop, UV_RUN_DEFAULT);
